src/WebUI/wwwroot/index.html

```diff
@@ -43,6 +43,14 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+
+    <meta Name="keywords" Content="GPT, ChatGPT, OpenAI, SSW, SSW Rules, Rule, Rules" />
+    <meta Property="og:title" Content="SSW RulesGPT" />
+    <meta Name="twitter:title" Content="SSW RulesGPT" />
+    <meta Name="description" Content="SSWBot is a helpful and informative GPT bot designed to assist users with rules related queries. " />
+    <meta Property="og:description" Content="SSWBot is a helpful and informative GPT bot designed to assist users with rules related queries. " />
+    <meta Name="twitter:description" Content="SSWBot is a helpful and informative GPT bot designed to assist users with rules related queries. go" />
```

Comment:
```suggestion
    <meta Name="twitter:description" Content="SSWBot is a helpful and informative GPT bot designed to assist users with rules related queries." />
```

Was the `go` bit intentional?

Reply:
Negative